underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -93,15 +93,19 @@ public HdfsUnderFileSystem(AlluxioURI ufsUri, UnderFileSystemConfiguration conf,
       Configuration hdfsConf) {
     super(ufsUri, conf);
     mUfsConf = conf;
+    ClassLoader previousClassLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
can you put a comment here to explain the reason:
```
// UserGroupInformation.setConfiguration(hdfsConf) will trigger service loading.
// Stash the classloader to prevent service loading throwing exception due to classloader mismatch
``

Reply:
Done.